.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,17 +1,16 @@
 ---
```

Comment:
@shreyasingh7673 Welcome to open source world !! You don't have to change in `feature_request.md` file. You have to edit it in Pull Request message.

Reply:
i got it. Do I need to send PR again?

Reply:
@Vishal-raj-1